.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,279 @@
+            -F access_token=$ROLLBAR_POST_SERVER_ITEM_TOKEN \
+            -F version=${{ github.sha }}.ios \
+            -F minified_url=http://reactnativehost/main.jsbundle \
+            -F source_map=@sourcemap.ios.js
+
+  android-codepush:
+    runs-on: ubuntu-latest
+    needs: [test, eslint, typescript, prettier]
+    if: github.env.SHOULD_DEPLOY
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - uses: egordm/gha-yarn-node-cache@v1
+      - name: 📦 Install Dependencies
+        run: yarn
+      - name: 📈 Run GraphQL Codegen
+        run: |
+          yarn gql:codegen
+          yarn gql:schema
+      - name: ⛅ OneSky Download
+        env:
+          ONESKY_API_KEY: ${{ secrets.ONESKY_API_KEY }}
+          ONESKY_SECRET_KEY: ${{ secrets.ONESKY_SECRET_KEY }}
+        run: yarn onesky:download
+      - name: 🏗️🚀🤖 Build and Codepush Android
+        env:
+          APPCENTER_TOKEN: ${{ secrets.APPCENTER_TOKEN }}
+        run: yarn codepush-android-staging --token "$APPCENTER_TOKEN" --sourcemap-output sourcemap.android.js --extra-bundler-option="--sourcemap-sources-root" --extra-bundler-option="./" --description "#${{ github.run_id }} ${{ github.sha }} | ${{ github.event.head_commit.message }}"
+      - name: 🔼🤖🗺️ Upload Android sourcemaps to Rollbar
+        env:
+          ROLLBAR_POST_SERVER_ITEM_TOKEN: ${{ secrets.ROLLBAR_POST_SERVER_ITEM_TOKEN }}
+        run: |
+          curl https://api.rollbar.com/api/1/sourcemap \
+            -F access_token=$ROLLBAR_POST_SERVER_ITEM_TOKEN \
+            -F version=${{ github.sha }}.android \
+            -F minified_url=http://reactnativehost/main.jsbundle \
+            -F source_map=@sourcemap.android.js
+
+  ios-deploy:
+    runs-on: macos-11.0
+    needs: [test, eslint, typescript, prettier]
+    if: github.env.SHOULD_DEPLOY
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - uses: egordm/gha-yarn-node-cache@v1
+      - name: 📦 Install Dependencies
+        run: yarn
+      - name: 📈 Run GraphQL Codegen
+        run: |
+          yarn gql:codegen
+          yarn gql:schema
+      - name: ⛅ OneSky Download
+        run: yarn onesky:download
+      - name: 📦💎 Bundle install
+        working-directory: ./ios
+        run: bundler install
+      - name: ➕ Add Commit SHA to ENVFILE
+        run: echo "COMMIT_SHA=$GITHUB_SHA" >> ./.env.production
+      - name: 🗝 Configure GitHub login
+        run: echo -e "machine github.com\n login $CI_USER_TOKEN" >> ~/.netrc
+      - name: 🏗️🚀🍏 Build and push to Testflight
+        env:
+          TRAVIS_BRANCH: ${{ github.ref }}
+          TRAVIS_TAG: ${{ github.ref }}
+          TRAVIS_COMMIT_MESSAGE: ${{ github.event.head_commit.message }}
+          TRAVIS_BUILD_NUMBER: ${{ github.run_id }}
+          MATCH_KEYCHAIN_NAME: ${{ secrets.MATCH_KEYCHAIN_NAME }}
```

Comment:
this can be moved to a constant value defined in https://github.com/CruGlobal/missionhub-react-native/blob/develop/ios/.env.default

There is nothing sensitive about the value: `MATCH_KEYCHAIN_NAME=MissionHub`